package.json

```diff
@@ -41,8 +41,8 @@
     "@thirdweb-dev/chain-icons": "^1.0.5",
     "@thirdweb-dev/chains": "^0.1.7-nightly-20230310185406-71532e5a",
     "@thirdweb-dev/react": "^3.10.4-nightly-20230310185406-71532e5a",
-    "@thirdweb-dev/sdk": "^3.10.4-nightly-20230310185406-71532e5a",
-    "@thirdweb-dev/storage": "^1.1.1-nightly-20230310185406-71532e5a",
+    "@thirdweb-dev/sdk": "4.0.0-dev-20230313143117-bcef98b",
```

Comment:
check your changeset - shouldnt be a major version for this.

also let's keep sdk/react in sync